### PR TITLE
Implement Kryptono ticker, order book and trades

### PR DESCRIPTION
**Pull request by Swingcloud**
- What is the purpose of this Pull Request? as title
- What is the related issue for this Pull Request? #549
- [x] I have added Specs
- [x] (If implementing Market Ticker) I have verified that the `volume` refers to BASE
- [x] (If implementing Market Ticker) I have verified that the `base` and `target` is assigned correctly
 look in to Kryptono exchange https://kryptono.exchange/k/accounts/trade?s=EOS_BTC
and see the endpoint output https://engine.kryptono.exchange/api/v1/cs?symbol=ETH_BTC&tt=1m

ticker on `EOS_BTC`
```ruby
# [
#  1527397440000, timestamp
#  "0.07971168",       open
#  "0.07971168",       high
#  "0.07971168",       low
#  "0.07971168",       close
#  "0.00000000"      volume
# ]
```
as I remember we always take 24h volume as base vol so i fetch 
https://engine.kryptono.exchange/api/v1/cs?symbol=ETH_BTC&tt=1d to get the volume
or like WCX fetch https://engine.kryptono.exchange/api/v1/cs?symbol=ETH_BTC&tt=4h and grab last 6's volume?

order book
```ruby
{
 "symbol": "EOS_BTC",
 "asks": [
  [
   "0.00170402",
   "0.44000000"
  ]
 ],
 "bids": [
  [
   "0.00158056",
   "0.25000000"
  ]
 ] 
```

trades 
```ruby
{
 "symbol": "EOS_BTC",
 "limit": 100,
 "history": [
  {
   "id": 453817,
   "price": "0.00162482",
   "qty": "0.28000000",
   "isBuyerMaker": true,
   "time": 1527292483227
  }
 ]
}
```

### CODE REVIEW

**Comment by tmlee** [comment]
@Swingcloud Using 4h and grab 6 would be more accurate i believe, as the 1d wont be last 24 hours but within the start of the day until current

File: README.md [hunk 1]
```diff
@@ -116,19 +116,20 @@ Or install it yourself as:
...
-| Jubi              | Y       |            |         |         | Y           |
-| Koinok            | Y       |            |         |         | Y           |
 | Koinex            | Y       |            |         |         | Y           |
+| Koinok            | Y       |            |         |         | Y           |
 | Korbit            | Y       |            |         |         | User-Defined|
 | Kraken            | Y       |            |         |         | Y           |
+| Kryptono          | Y       | Y          | Y       |         | User-Defined|
 | Kucoin            | Y       | Y          | Y       |         | Y           |
 | Kuna              | Y       | Y          | Y       |         | User-Defined|
+| KyberNetwork      | Y       | N          | N       |         | Y           |
```
Comment (tmlee) [inline README.md]: Why are there changes for these lines?

File: lib/cryptoexchange/exchanges/kryptono/services/market.rb [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            true
+          end
+        end
+
+        def fetch(market_pair)
+          ticker_output = super(ticker_url(market_pair, "1m"))
+          volume_output = super(ticker_url(market_pair, "1d"))
```
Comment (tmlee) [inline lib/cryptoexchange/exchanges/kryptono/services/market.rb]: 4h, grab 6 would be more accurate i believe

File: README.md (hunk 1 above)
Comment (Swingcloud) [reply README.md]: IIRC this table is list for alphabetical orders, but seems a little messy in letter K, so I change the order

**Pull request merged**